pyvista/core/composite.py

```diff
@@ -61,6 +61,7 @@ class MultiBlock(vtkMultiBlockDataSet, CompositeFilters, DataObject):
     # Bind pyvista.plotting.plot to the object
     plot = pyvista.plot
     _READERS = dict.fromkeys(['.vtm', '.vtmb'], vtk.vtkXMLMultiBlockDataReader)
+    _READERS['.case'] = vtk.vtkGenericEnSightReader
```

Comment:
@JevinJ, is this how you intended this to be done?

Reply:
@JevinJ, should this reader be added here: https://github.com/pyvista/pyvista/blob/24d572e5b64756e14fa632ed59e982cfd9acebe1/pyvista/utilities/fileio.py#L12

Reply:
I'm not sure the code in the first one will even run. You can just replace fromkeys with a more explicit declaration, grommets was just simpler since there was only 1 value being used for all keys. I'll have a more thorough look tomorrow.

Reply:
So just do:

    _READERS = {'.vtm':  vtk.vtkXMLMultiBlockDataReader, '.vtmb': vtk.vtkXMLMultiBlockDataReader,
                         '.case': vtk.vtkGenericEnSightReader}